gdsfactory/cell.py

```diff
@@ -311,6 +294,11 @@ def wrapper(*args, **kwargs) -> Component:
             component.module = func.__module__
             component.__doc__ = func.__doc__
 
+        for post in post_process or []:
+            component = post(component)
```

Comment:
Is this now the same as the `decorator` argument that we had before?

Reply:
yes, but this one is applied as 

```
@gf.cell(post_process=[add_label, add_marker_layer])
def straight_pdk_a(**kwargs)
   return gf.components.straight(**kwargs)

```